### DISCUSSION TITLE
clipping path recognition

### DISCUSSION THREAD

**Discussion by gent99** [ImageMagick/ImageMagick#5482] [Help]
Hi,

we're using this version:

Version: ImageMagick 6.9.10-69 Q16 x86_64 2019-10-26 https://imagemagick.org
Copyright: © 1999-2019 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP(3.1) 
Delegates (built-in): bzlib cairo djvu fftw fontconfig freetype gslib gvc heic jbig jng jp2 jpeg lcms ltdl lzma openexr pangocairo png ps raqm raw rsvg tiff webp wmf x xml zlib

and we see, that clipping path is not recognized. On an older version Version: ImageMagick 6.7.2-7 the clipping path results in a white background. With 6.9.10, the clipping path is not recognized, so the image has its normal background.

What can we do?

Kind regards Holger

**Comment by snibgo** [comment]
> What can we do?

You can report an issue in https://github.com/ImageMagick/ImageMagick/issues

You will need to upload an image file with clipping path that your 6.9.10-69 ignores, but 6.7.2-7 regards.

And, of course, show the exact commands you used.
